backend/todo-list-backend/src/main/java/com/codesquad/esfj/todolist/task/Task.java

```diff
@@ -0,0 +1,131 @@
+package com.codesquad.esfj.todolist.task;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.util.StringUtils;
+
+import java.util.Objects;
+
+public class Task {
+    public static final long TOP_PREVIOUS_ID = -1L;
+
+    @Id
+    private Long id;
+    private String title;
+    private String content;
+    private String writer;
+    private boolean deleted;
```

Comment:
```suggestion
    private boolean isDeleted;
```

boolean 형 값의 변수명은 이런 형태로 짓는 습관을 들여주세요.